src/ts/ui/runner/runui.ts

```diff
@@ -434,14 +445,32 @@ export class RunUI extends UIScreen {
         sampleIndex = this.treeScrubber.showLatestBaseTree ? UNSET : this.treeScrubber.sampledIndex,
         {muHist, muStarHist, totalBranchLengthHist, logPosteriorHist, numMutationsHist, popGHist, kneeIndex} = this.pythia;
       this.treeScrubber.setData(last, kneeIndex, mccIndex);
+
+      const muud = muHist.map(n=>n*MU_FACTOR);
+      const totalLengthYear = totalBranchLengthHist.map(t=>t/DAYS_PER_YEAR);
+      const popHistGrowth = popGHist.map(g=>POP_GROWTH_FACTOR/g);
+      const serieses = [logPosteriorHist, muud,
+        totalLengthYear, numMutationsHist, popHistGrowth];
       this.logPosteriorCanvas.setData(logPosteriorHist, kneeIndex, mccIndex, hideBurnIn, sampleIndex);
-      this.muCanvas.setData(muHist.map(n=>n*MU_FACTOR), kneeIndex, mccIndex, hideBurnIn, sampleIndex);
+      this.muCanvas.setData(muud, kneeIndex, mccIndex, hideBurnIn, sampleIndex);
       if (this.isApobecEnabled) {
-        this.muStarCanvas.setData(muStarHist.map(n=>n*MU_FACTOR), kneeIndex, mccIndex, hideBurnIn, sampleIndex);
+        const muudStar = muStarHist.map(n=>n*MU_FACTOR);
+        this.muStarCanvas.setData(muudStar, kneeIndex, mccIndex, hideBurnIn, sampleIndex);
+        serieses.push(muudStar);
       }
-      this.TCanvas.setData(totalBranchLengthHist.map(t=>t/DAYS_PER_YEAR), kneeIndex, mccIndex, hideBurnIn, sampleIndex);
+      this.TCanvas.setData(totalLengthYear, kneeIndex, mccIndex, hideBurnIn, sampleIndex);
       this.mutCountCanvas.setData(numMutationsHist, kneeIndex, mccIndex, hideBurnIn, sampleIndex);
-      this.popGrowthCanvas.setData(popGHist.map(g=>POP_GROWTH_FACTOR/g), kneeIndex, mccIndex, hideBurnIn, sampleIndex);
+      this.popGrowthCanvas.setData(popHistGrowth, kneeIndex, mccIndex, hideBurnIn, sampleIndex);
+
+      if (this.pythia && this.pythia.kneeIndex <= 0) {
```

Comment:
It would be even better to keep adjusting the proposed knee index as long as the user hasn't specified one explicitly.  Here's a screenshot of how the current setup doesn't make a great guess to begin with, but later has enough information to make a better guess (I ran the demo and set "Steps per sample" to 100,000; the cursor is around where I would have placed the burn-in cutoff manually)
![Screenshot from 2024-10-22 09-46-43](https://github.com/user-attachments/assets/0427e055-6bfb-4f3b-9cc6-6480371d0fb9)

Reply:
Trickier, but I'll take a peek.

Reply:
hrm, if I never override the auto suggested knee, should we export the kneeIndex? Because if I reload it, currently there would be no way to distinguish between the auto-selected knee and a curated knee. We can do a bunch of juggling to unset the knee on export and then reset it, and auto set the knee on load, or we can add a flag to the export for something like `kneeIsManuallySet` or `kneeIsCurated` ?

Reply:
Hrm, consistently updating the knee from the latest sample results in only a few sampled trees. Here's a long run of 1777 samples, and you can see the knee allows only 4: 
<img width="1622" alt="Screen Shot 2024-10-22 at 9 26 44 AM" src="https://github.com/user-attachments/assets/f3be0547-6f32-4dbd-b294-d56be691e2ca">

Exporting and reloading the file with the the acceptable range at 3 std deviations rather than just 2, we get 53 samples, at 4 std dev we get 56, 5 -> 1312. 

Hrm, if we repeatedly calculate the knee, then I retract my comment about needing to set the limit at 2 std dev. That's too restrictive, at least for this data set. 5 doesn't look that bad while in progress: 
<img width="1622" alt="Screen Shot 2024-10-22 at 9 53 57 AM" src="https://github.com/user-attachments/assets/fb7b5184-265a-45d2-adcc-7771ad25e12d">

@patvarilly what do you think? 

N.B. still have to work out the mechanics of distinguishing between the auto and user selected knee.